server/router/ride.ts

```diff
@@ -276,20 +258,23 @@ router.put('/:id/edits', validateUser('User'), (req, res) => {
         // create replace edit and add replaceId to edits field
         db.create(res, replaceRide, (editRide) => {
           db.update(res, Ride, { id }, addEditOperation, tableName, () => {
-            res.send(editRide);
+            notifyEdit(editRide, change, userType, userId)
+              .then(() => res.send(editRide)).catch(() => res.send(editRide));
+            // res.send(editRide);
           });
         });
       } else {
-        res.send(ride);
+        notifyEdit(ride, change, userType, userId)
+          .then(() => res.send(ride)).catch(() => res.send(ride));
```

Comment:
this case doesn't need a notification, as this is technically seen as removing an unscheduled ride

Reply:
Updated with changes